tests/testthat/test-multilevelannotationstep2.R

```diff
@@ -1,79 +1,85 @@
+    load(file = file.path(test_path, "tempobjects.Rda"))
 
-        expected <- readRDS("chemscoremat.Rds")
-        load(file = "step1_results.Rda")
-        load(file = "global_cor.Rda")
-        load(file = "tempobjects.Rda")
+    outloc <- file.path(
+      tempdir(),
+      "multilevelannotationstep2",
+      test_identifier
+    )
+    dir.create(outloc, recursive = TRUE)
 
-        # Act
-        actual <- lapply(
-            1:num_sets,
-            call_multilevelannotationstep2,
-            outloc = tempdir(),
-            max.rt.diff = max.rt.diff,
-            chemids_split = chemids_split,
-            num_sets = num_sets,
-            mchemdata = mchemdata,
-            mass_defect_window = mass_defect_window,
-            corthresh = corthresh,
-            global_cor = global_cor,
-            mzid = mzid,
-            adduct_table = adduct_table,
-            adduct_weights = adduct_weights,
-            max_isp = max_isp,
-            MplusH.abundance.ratio.check = MplusH.abundance.ratio.check,
-            mass_defect_mode = mass_defect_mode,
-            chemids = chemids,
-            isop_res_md = isop_res_md,
-            filter.by = filter.by
-        )
-        actual <- ldply(actual, rbind)
-        actual$Formula<-gsub(actual$Formula,pattern="_.*",replacement="")
+    # Act
+    actual <- lapply(
+      1:num_sets,
+      call_multilevelannotationstep2,
+      outloc = outloc,
+      max.rt.diff = max.rt.diff,
+      chemids_split = chemids_split,
+      num_sets = num_sets,
+      mchemdata = mchemdata,
+      mass_defect_window = mass_defect_window,
+      corthresh = corthresh,
+      global_cor = global_cor,
+      mzid = mzid,
+      adduct_table = adduct_table,
+      adduct_weights = adduct_weights,
+      max_isp = max_isp,
+      MplusH.abundance.ratio.check = MplusH.abundance.ratio.check,
+      mass_defect_mode = mass_defect_mode,
+      chemids = chemids,
+      isop_res_md = isop_res_md,
+      filter.by = filter.by
+    )
+    actual <- ldply(actual, rbind)
+    actual$Formula <- gsub(actual$Formula, pattern = "_.*", replacement = "")
 
 
-        keys <- c('mz', 'time', 'Name', 'Adduct', 'Formula', 'chemical_ID', 'cur_chem_score')
+    keys <- c("mz", "time", "Name", "Adduct", "Formula", "chemical_ID", "cur_chem_score")
 
-        actual <- dplyr::arrange_at(
-          actual, keys
-        )
-        expected <- dplyr::arrange_at(
-          expected, keys
-        )
+    actual <- dplyr::arrange_at(
+      actual, keys
+    )
+    expected <- dplyr::arrange_at(
+      expected, keys
+    )
 
-        comparison <- dataCompareR::rCompare(actual, expected, keys = keys)
-        dataCompareR::saveReport(
-            comparison,
-            reportName = test_identifier,
-            reportLocation = tempdir(),
-            showInViewer = FALSE,
-            mismatchCount = 1000
-        )
+    comparison <- dataCompareR::rCompare(actual, expected, keys = keys)
```

Comment:
Is `dataCompareR` conda friendly now?

Reply:
it seems to have a package https://anaconda.org/conda-forge/r-datacomparer

Reply:
yeah but @hechth mentioned there was an issue with that